plugins/confluent/credentials.go

```diff
@@ -0,0 +1,94 @@
+package confluent
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+const (
+	MDS_URL                      = sdk.FieldName("MDS URL")
```

Comment:
I'd propose we also move this one to `fieldname/names.go` for consistency with other plugins. We could also consider renaming this to `URL`, since it contains the secret that would be passed via `--url` to `confluent login`, but I'll defer this decision to you.

Reply:
I debated on just using URL, but since I couldn't complete the work for the Platform side of things I just left it alone. Can just move this to URL and rely on the Description to declare what URL we are talking about.